app/src/pointing/input_processor_temp_layer.c

```diff
@@ -134,9 +134,6 @@ static int handle_layer_state_changed(const struct device *dev, const zmk_event_
     if (ret < 0) {
         return ret;
     }
-    if (data->state.toggle_layer == 0) {
-        return ZMK_EV_EVENT_BUBBLE;
-    }
```

Comment:
Special handling looks not required, since `zmk_keymap_layer_active` properly handles default layer.

Reply:
Is the main issue that the mutex is stuck if we hit this branch, since it `return`s without unlocking?

Reply:
Yeah, this seems reasonable. Especially since we may *change* what is the default/"base" layer later, and don't really want a special check here.